### ISSUE TITLE
Image Preview of the Themes

### ISSUE THREAD

**Issue by enesbala5** [jeslie0/nixos-grub-themes#3]
I think it would be very helpful if we could have an idea of what each theme looks like before installing it to our system.

Thanks in advance!

**Comment by jeslie0** [comment]
Agreed, it is useful to have previews. I'm not sure of programmatic way to do that though. We might not need one as it should be simple to do it by hand. I will have a think about it and see what I can do.

**Comment by enesbala5** [comment]
@jeslie0 Thank you!

**Comment by jeslie0** [comment]
I think the way to go is to to have a static site that displays images of the available themes. I don't want to bloat up this repository with image files and sadly I don't think I am able to properly customize images linked in the README. I have a nice idea for making such a site and will post back when it is done.
